_plugins/modal-details-lib/modal-details.rb

```diff
@@ -0,0 +1,38 @@
+require 'erb'
+
+module Jekyll
+    module ModalDetails
```

Comment:
Is this one still used?

Reply:
it has not been used but I thought why not add it still - its isolated to the commit https://github.com/scala/docs.scala-lang/pull/2414/commits/d0b1da04f7717e89995d82060be20a334f0fd839

Reply:
I dropped it